shared/chatTypes.d.ts

```diff
@@ -1,27 +1,28 @@
-import { User, UserId } from "./userTypes.js";
-export type MessageId = string;
+import { User, UserId } from "./userTypes.js"
```

Comment:
I think this and all the other 'generated' files should not be in git; I'm not sure how the shared lib should be configured exactly, but this definitely feels like something that should be .gitignored (also it's strange that these files are generated in the module root...feels like there should be a dist folder for this or something).  Not 100% sure how to fix

Reply:
Yes. It should normally be in dist. But doing that means we have to change all the import path.

shared/wsTypes.js will be shared/dist/wsTypes.

When installing from npm, the user will import files specified as the entry point in package.json which can be set to the dist folder.

But using our filesystem method, it doesn't work that way. The importing just work normally, so if we want to import modules in dist, we have to import from shared/dist